pkg/realserver/configure.go

```diff
@@ -485,6 +642,39 @@ func (r *realserver) setAddresses() error {
 	return nil
 }
 
+func (r *realserver) setAddresses6() error {
```

Comment:
how different is this from setAddresses?

Reply:
- it `gets` from the ip interface only v6 addresses to create the `removals` set
- it only checks to add config6 addrs

Updated the interface method `(i *ipManager) Get() ([]string, error)`  to `(i *ipManager) Get(v4, v6 bool) ([]string, error)`.

The args passed to the child function `(i *ipManager) get(ctx context.Context, IPv4, IPv6 bool) ([]string, error)`, which was previously being passed with hard-coded values `true, false`

So they separate concerns about removing and setting. Granted they are pretty similar but I like having the two calls separate and explicit. The caller can easily tell what they do which is nice

Reply:
it might be difficult to reason about the caller.  Absent any other context, what's the behavior of`Get(true, false)` versus `Get(false,false)` versus `Get(true, true)` ? Might be an improvement to always return a pair of slices.

`func Get() (v4 []string, v6 []string, error)`

Reply:
done